js/components/portal-dashboard/response-details/popup-question-answer-list.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { Map } from "immutable";
+import Answer from "../../../containers/portal-dashboard/answer";
+import { renderHTML } from "../../../util/render-html";
+
+import css from "../../../../css/portal-dashboard/response-details/popup-student-response-list.less";
+
+interface IProps {
+  activities: Map<any, any>;
+  currentActivity: Map<string, any>;
+  currentStudentId: string | null;
+  students: Map<any, any>;
+}
+
+export class PopupQuestionAnswerList extends React.PureComponent<IProps> {
+  render() {
+    const { activities, currentStudentId, students, currentActivity } = this.props;
+    const currentActivityId = currentActivity?.get("id");
+    const student = currentStudentId ? students.toArray().find((s: any) => s.get("id") === currentStudentId)
+      : students.toArray()[0];
+    const activity = currentActivityId ? activities.toArray().find((a: any) => a.get("id") === currentActivityId)
+      : activities.toArray()[0];
```

Comment:
These two might be a little cleaner as:
```
    const student = currentStudentId 
                    ? students.toArray().find((s: any) => s.get("id") === currentStudentId)
                    : students.toArray()[0];
    const activity = currentActivityId 
                     ? activities.toArray().find((a: any) => a.get("id") === currentActivityId)
                     : activities.toArray()[0];
                     
```
lining up the "?" and ":" tends to be the more common style practice.